src/reducers/dependencies.reducer.test.js

```diff
@@ -0,0 +1,287 @@
+import reducer, { getDependenciesForProjectId } from './dependencies.reducer';
+import {
+  LOAD_DEPENDENCY_INFO_FROM_DISK,
+  ADD_DEPENDENCY_START,
+  ADD_DEPENDENCY_ERROR,
+  ADD_DEPENDENCY_FINISH,
+  UPDATE_DEPENDENCY_START,
+  UPDATE_DEPENDENCY_ERROR,
+  UPDATE_DEPENDENCY_FINISH,
+  DELETE_DEPENDENCY_START,
+  DELETE_DEPENDENCY_ERROR,
+  DELETE_DEPENDENCY_FINISH,
+} from '../actions';
+import { isIterable } from 'core-js';
+
+describe('dependencies reducer', () => {
+  it('should return initial state', () => {
+    expect(reducer(undefined, {})).toEqual({});
+  });
+  it(`should handle ${LOAD_DEPENDENCY_INFO_FROM_DISK}`, () => {
+    const prevState = {
+      baz: {},
+    };
+    const action = {
+      type: LOAD_DEPENDENCY_INFO_FROM_DISK,
+      projectId: 'foo',
+      dependencies: { redux: {} },
+    };
+    expect(reducer(prevState, action)).toMatchSnapshot();
```

Comment:
Interesting! I've never thought to use snapshot tests for Redux reducers.

My knee-jerk reaction is that it's a bit more opaque, since the comparison is to something in another file and not so easy to co-locate to compare. There's something really nice about `expect(object1).toEqual(object2)`.

But, the more I think about it, the more I'm coming around on the idea... it means that rewriting the reducers wouldn't mean a big timesink updating the tests. I'd worry a bit about someone updating the snapshots and missing a subtle thing that a proper test would have made more explicit, but presumably we'd catch that in review, the same way we would if someone modified the tests.

At any rate, even if I think standard comparison is a better long-term solution, this is a brilliant way to get quick test coverage. We can always revisit later, but for now I'm delighted that we have this :D

Reply:
If you want to keep the snapshots in the same file, there’s [`toMatchInlineSnapshot`](https://jestjs.io/docs/en/snapshot-testing#inline-snapshots), which requires that you use Prettier, but embeds the snapshot template string directly into the test.

Reply:
Oh wow! That sounds amazing.

@mik639 I don't wanna burden you with more changes - if you don't have the bandwidth, don't worry about it at all - but if you wanted to experiment with `toMatchInlineSnapshot` I think it could be really neat!

Reply:
Why not, I will change